### ISSUE TITLE
When Ursula is inited with `--federated-only`, this option is still required when you run Ursula

### ISSUE THREAD

**Issue by michwill** [nucypher/nucypher#711]
```
nucypher ursula init --federated-only     

                                  _               
                                 | |              
     _ __  _   _  ___ _   _ _ __ | |__   ___ _ __ 
    | '_ \| | | |/ __| | | | '_ \| '_ \ / _ \ '__|
    | | | | |_| | (__| |_| | |_) | | | |  __/ |   
    |_| |_|\__,_|\___|\__, | .__/|_| |_|\___|_|   
                       __/ | |                    
                      |___/|_|  

    version 0.1.0-alpha.4


WARNING: Running in Federated mode
Enter Ursula's public-facing IPv4 address: <censored>                       

Enter keyring password: 
Repeat for confirmation: 
Generated keyring /home/michwill/.local/share/nucypher/keyring
Saved configuration file /home/michwill/.local/share/nucypher/ursula.config

To run an Ursula node from the default configuration filepath run: 

'nucypher ursula run'
```
```
nucypher ursula run --network <REDACTED> --teacher-uri ...

                                  _               
                                 | |              
     _ __  _   _  ___ _   _ _ __ | |__   ___ _ __ 
    | '_ \| | | |/ __| | | | '_ \| '_ \ / _ \ '__|
    | | | | |_| | (__| |_| | |_) | | | |  __/ |   
    |_| |_|\__,_|\___|\__, | .__/|_| |_|\___|_|   
                       __/ | |                    
                      |___/|_|  

    version 0.1.0-alpha.4


Decrypting keyring...
Enter keyring password: 
Traceback (most recent call last):
  File "/home/michwill/Projects/nucypher-node/.venv/bin/nucypher", line 10, in <module>
    sys.exit(nucypher_cli())
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/cli/main.py", line 394, in ursula
    node = Ursula.from_teacher_uri(teacher_uri=teacher_uri, min_stake=min_stake, federated_only=federated_only)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/characters/lawful.py", line 702, in from_teacher_uri
    return __attempt()
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/characters/lawful.py", line 692, in __attempt
    minimum_stake=min_stake)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/utilities/decorators.py", line 45, in wrapped
    return func(*args, **kwargs)  # ... nothing to validate
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/characters/lawful.py", line 759, in from_seed_and_stake_info
    certificate_filepath=certificate_filepath)
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/network/nodes.py", line 899, in verify_node
    self.validate_metadata(accept_federated_only)  # This is both the stamp and interface check.
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/network/nodes.py", line 877, in validate_metadata
    self.stamp_is_valid()
  File "/home/michwill/Projects/nucypher-node/.venv/lib/python3.6/site-packages/nucypher/network/nodes.py", line 863, in stamp_is_valid
    raise self.WrongMode(message)
nucypher.network.nodes.WrongMode: This node can't be verified in this manner, but is OK to use in federated mode if you have reason to believe it is trustworthy.
Sentry is attempting to send 1 pending error messages
Waiting up to 2.0 seconds
Press Ctrl-C to quit
```

**Comment by tuxxy** [comment]
Related to https://sentry.io/nucypher/nucypher/issues/863851541/

**Comment by KPrasch** [comment]
Closed in #712
